lib/parser/source/tree_rewriter/action.rb

```diff
@@ -46,10 +46,52 @@ def ordered_replacements
         reps
       end
 
+      def nested_actions
+        reps = []
```

Comment:
I guess `reps` stands for `replacements` from the method above. Maybe let's rename it so it makes more sense? Like `actions` or `result`

Reply:
Indeed